mw-webapp/src/logic/allWaysPage/AllWaysPage.module.scss

```diff
@@ -0,0 +1,17 @@
+@import "src/styles/variables.scss";
+
+.titlePage {
+  position: absolute;
+  top: 1rem;
+  left: 47%;
+  color: $lightyellow;
+  text-shadow: 2px 2px 2px $button-shadow;
+}
+
+.titleWhenLoggedIn {
+  position: absolute;
+  top: 2.2rem;
+  left: 47%;
+  color: $lightyellow;
+  text-shadow: 2px 2px 2px $button-shadow;
+}
```

Comment:
Will you use these styles for title on each of 8 pages?

Reply:
I only know about three pages, I made mixins